common/src/autogluon/common/utils/utils.py

```diff
@@ -61,8 +61,9 @@ def setup_outputdir(path, warn_if_exist=True, create_dir=True, path_suffix=None)
             logger.warning(
                 f'Warning: path already exists! This predictor may overwrite an existing predictor! path="{path}"'
             )
-    path = os.path.expanduser(path)  # replace ~ with absolute path if it exists
-    path = os.path.abspath(path)
+    if not path.lower().startswith("s3://"):
```

Comment:
If we plan to support s3, then we should also edit the code above this when `warn_if_exist` to not call `os.makedirs` when the path is an s3 path.

Reply:
Also, if we plan to add support for s3, lets add a short docstring that mentions this